### ISSUE TITLE
NVDA fail to bring mouse pointer to the cursor navigator object with nvda+numpad divide after one time.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2349]
**Reported by aleskis on 2012-05-16 13:07**
To reproduce
1. With Firefox go to:
https://mobile.free.fr/moncompte/
2. Type B to go on button, then arrow up one time. You are on virtual numpad.
3. Bring mouse pointer on the first clickable position with numpad Divide. First time it okay.
4. Type G to go to the next position on the numpad and numpad divide a second time. In this second case, mouse pointer fail to follow cursor navigator.

**Comment by ehollig** [comment]
When I go to the requested URL and press B, I am placed on the "Annuler" button. If I then press G, there is no graphic found. I am able to navigate to any other graphic on this page and route the mouse to the review cursor without a problem.

**Comment by jcsteh** [comment]
The exact instructions don't get me to the graphic either, though I was able to find the numeric pad. However, routing the mouse to all of these graphics works for me too.
